plugins/default/lib/test_mongodb_health.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+import unittest
+
+from mongodb_health import analize, parse_mongo_result, report
+
+
+class TestAnalize(unittest.TestCase):
+
+    def runTest(self):
+        self.assertEqual(analize(1), 0)
+        self.assertEqual(analize(2), 0)
+        self.assertEqual(analize(0), 1)
+        self.assertEqual(analize(7), 1)
+        self.assertEqual(analize(3), 2)
+        self.assertEqual(analize(5), 2)
+        self.assertEqual(analize(7), 1)
+        self.assertEqual(analize(9), 2)
+        self.assertEqual(analize(None), 3)
+
+
+class TestParseMongoResult(unittest.TestCase):
+
+    def runTest(self):
+        self.assertEqual(parse_mongo_result("0\n"), 0)
+        self.assertEqual(parse_mongo_result("1\n"), 1)
+        self.assertEqual(parse_mongo_result("2\n"), 2)
+        self.assertEqual(parse_mongo_result("3\n"), 3)
+        self.assertEqual(parse_mongo_result("5\n"), 5)
+        self.assertEqual(parse_mongo_result("7\n"), 7)
+        self.assertEqual(parse_mongo_result("9\n"), 9)
+        self.assertEqual(parse_mongo_result("10\n"), 10)
+        self.assertEqual(parse_mongo_result("\n"), None)
+        self.assertEqual(parse_mongo_result(""), None)
+
+
+class TestReport(unittest.TestCase):
+
+    def runTest(self):
+        self.assertEqual(report(
+            ("mongodb-1-1-s1ngl",),
+            (1,),
+            (0,)), 0)
+        self.assertEqual(report(
+            ("mongodb-1-1-9hj1x", "mongodb-2-1-xu70c", "mongodb-3-1-wdq55"),
+            (1, 2, 2),
+            (0, 0, 0)), 0)
+        self.assertEqual(report(
+            ("mongodb-1-1-9hj1x", "mongodb-2-1-xu70c", "mongodb-3-1-wdq55"),
+            (1, 2, 1),
+            (0, 0, 0)), 2)
+        self.assertEqual(report(
+            ("mongodb-1-1-9hj1x", "mongodb-2-1-xu70c"),
+            (1, 2),
+            (0, 0)), 1)
+        self.assertEqual(report(
+            ("mongodb-1-1-9hj1x", "mongodb-2-1-xu70c", "mongodb-3-1-wdq55"),
+            (0, 2, 2),
+            (1, 0, 0)), 2)
+        self.assertEqual(report(
+            ("mongodb-1-1-9hj1x", "mongodb-2-1-xu70c", "mongodb-3-1-wdq55"),
+            (1, 7, 2),
+            (0, 1, 0)), 1)
+        self.assertEqual(report((), (), ()), 3)
```

Comment:
Again, please use constants and no magic numbers... I suspect `3` is really `nagios.UNKNOWN`.
Imagine you or someone else looking at a test failure of this test... and you see the error message, you see something like `2 != 3`, and you have no clue what's going on.

We should strive to make the tests legible for new people coming into the code base, not require them to imagine that Nagios use 3 to mean unknown exit code.